pygmt/helpers/decorators.py

```diff
@@ -673,14 +674,19 @@ def kwargs_to_strings(**conversions):
     >>> module(123, bla=(1, 2, 3), foo=True, A=False, i=(5, 6))
     {'A': False, 'bla': (1, 2, 3), 'foo': True, 'i': '5,6'}
     args: 123
+
+    >>> # Test that region accepts arguments with datetime or timedelta type
     >>> import datetime
     >>> module(
     ...     R=[
     ...         np.datetime64("2010-01-01T16:00:00"),
     ...         datetime.datetime(2020, 1, 1, 12, 23, 45),
+    ...         np.timedelta64(0, "h"),
+    ...         np.timedelta64(24, "h"),
     ...     ]
     ... )
-    {'R': '2010-01-01T16:00:00/2020-01-01T12:23:45.000000'}
+    {'R': '2010-01-01T16:00:00/2020-01-01T12:23:45.000000/0/24'}
```

Comment:
Apparently Python has a built-in [`datetime.timedelta`](https://docs.python.org/3/library/datetime.html#timedelta-objects) objects, but it's super hard to cast it to an integer type while preserving the orignal unit (e.g. day, hour, minute, etc). E.g.

```python
import datetime

td = datetime.timedelta(hours=24)

print(np.timedelta64(td))
# numpy.timedelta64(86400000000,'us')
print(np.timedelta64(td).astype(int))
# 86400000000
```

Should we still support `datetime.timedelta`? Or only `np.timedelta64`?